### PR TITLE
add recipe of views

### PR DESCRIPTION
**Pull request by knaufk**

### CODE REVIEW

File: recipes/13/13_views.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# 13 Encapsulating Logic with (Temporary) Views
+
+:bulb: This example will show how you can use (temporary) views to reuse code and to structure long queries and scripts. 
+
+`CREATE (TEMPORARY) VIEW` defines a view from a query. 
+**A view is not physically materialized.** 
+Instead, the query is run every time the view is referenced in a query.
+
+Temporary views are very useful to structure and decompose more complicated queries and to re-use queries within a longer script.
+Non-temporary views - stored in a persistent catalog - can also be used to share common queries within your organization, e.g. common filters or pre-processing steps.
```
Comment (morsapaes) [inline recipes/13/13_views.md:10]: Maybe link to the catalog documentation? We haven't introduced catalogs yet.

File: recipes/13/13_views.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+`CREATE (TEMPORARY) VIEW` defines a view from a query. 
+**A view is not physically materialized.** 
+Instead, the query is run every time the view is referenced in a query.
+
+Temporary views are very useful to structure and decompose more complicated queries and to re-use queries within a longer script.
+Non-temporary views - stored in a persistent catalog - can also be used to share common queries within your organization, e.g. common filters or pre-processing steps.  
+
+Here, we create a view on the `server_logs` that only contains successful requests. 
+This view encapsulates the logic of filtering the logs based on certain `status_code`s. 
+This logic subsequently be used by any query or script that has access to the catalog.
```
Comment (morsapaes) [inline recipes/13/13_views.md:14]:
```suggestion
This logic can subsequently be used by any query or script that has access to the catalog.   
```

**Review by morsapaes — COMMENTED** [review]
Looks good, just left a comment about linking to docs.

File: recipes/13/13_views.md (hunk 1 above)
Comment (knaufk) [reply recipes/13/13_views.md]: Done.

**Pull request merged**